diary-alexander-jaffray/modernized/32-ch-16-scotland.adoc

```diff
@@ -310,13 +321,21 @@ So, dear Friends, in a fresh stream of Life, does my soul dearly salute you all,
 desiring the continuance of your prayers for me: so,
 I remain your Friend and Brother in the Lord,
 
-Peter Gardiner.`"
+[.signed-section-signature]
+Peter Gardiner.
+
+[.postscript]
```

Comment:
Also here I guess isn't part of the epistle either. Maybe I missed why these were included in his actual epistle.